Source/Core/Core/Boot/Boot.cpp

```diff
@@ -272,7 +272,8 @@ bool CBoot::BootUp()
 
     const DiscIO::IVolume& pVolume = DVDInterface::GetVolume();
 
-    if ((pVolume.GetVolumeType() == DiscIO::Platform::WII_DISC) != _StartupPara.bWii)
+    if (pVolume.GetVolumeType() == DiscIO::Platform::WII_DISC &&
+        _StartupPara.m_system != SConfig::System::Wii)
```

Comment:
Isn't this supposed to be `(pVolume.GetVolumeType() == DiscIO::Platform::WII_DISC) != (_StartupPara.m_system == SConfig::System::Wii)` so that the case where the disc is GC and the console is Wii is caught? (Not that this panic alert really is useful right now...)